cmd/arcaflow/main.go

```diff
@@ -152,6 +135,9 @@ logged_outputs:
 			flag.Usage()
 			os.Exit(ExitCodeInvalidData)
 		}
+	} else {
+		// If no config file is passed, we use an empty map to accept the schema defaults
+		configData = make(map[string]any)
```

Comment:
I think the way Go is specified, a `nil` map should behave (on read) exactly like an empty map. (And may actually be the same thing internally.)

Which, I think, means this `else` shouldn't be necessary. (Although I wonder whether `configData` should be *typed* as a map rather than `any`.)

Reply:
I tried settings `configData = nil` and I got a type error.